### PR TITLE
[ON HOLD] Homepage Template

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #31 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
 -e git+https://github.com/CodeForAfricaLabs/HURUmap.git@develop#egg=hurumap
-gunicorn==19.8.1
```
Comment (DavidLemayian) [inline requirements.txt]: Please put this back.

File: README.md [hunk 2]
```diff
@@ -10,17 +10,29 @@ TODO
...
 ## Development
 
-## Development
+### Server Requirements
+You will need to make sure your server meets the following requirements:
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+- [pip](https://pip.pypa.io/en/stable/installing/)
+- python (2.7)
+- [yarn](https://yarnpkg.com/en/docs/install#debian-stable)
+- node
```
Comment (DavidLemayian) [inline README.md]: Please remove node and yarn as we don't use for this.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "https://github.com/TakwimuAfrica/TAKWIMU/issues"
+  },
+  "homepage": "https://github.com/TakwimuAfrica/TAKWIMU#readme",
+  "dependencies": {
+    "bootstrap": "^4.1.1",
+    "font-awesome": "^4.7.0",
+    "jquery": "^3.3.1",
+    "popper": "^1.0.1"
+  }
+}
```
Comment (DavidLemayian) [inline package.json]: Please delete as we aren't using yet.

File: takwimu/static/css/styles.css [hunk 4]
```diff
@@ -1,8 +1,45 @@
...
+	text-decoration:none;
+}
+
+.footer-logo {
+  max-height: 65px;
+  width: 100px; }
+
+ img {
+ 	max-width:100%;
+ }
```
Comment (DavidLemayian) [inline takwimu/static/css/styles.css]: Please check indentation.

File: takwimu/static/css/_custom.scss [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    width: 100%;
+    height: 48em;
+    position: relative;
+    border-bottom: 3px solid #336781;
+}
+
+
+
+
+
```
Comment (DavidLemayian) [inline takwimu/static/css/_custom.scss]: Please check indentation.

File: takwimu/urls.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+"""
+from django.conf.urls import url
+from django.contrib import admin
+from takwimu.views import HomeView
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^$', HomeView.as_view(), name='homepage'),
+
+]
```
Comment (DavidLemayian) [inline takwimu/urls.py]: @phillipahereza I thought you did this in another PR?

**Comment by mtuchi** [comment]
@DavidLemayian 
I think PR is ready for merge, Can you review and let me know what other changes are needed maybe ?

**Comment by DavidLemayian** [comment]
**TO MOVE TO CodeForAfricaLabs/HURUmap-apps at a later date.**

**Pull request closed without merging**